### PR TITLE
REST API tests for IAM

### PR DESCRIPTION
**Pull request by kirill-sizov**
### Motivation and context

Functionality that was made in PR #3788 has no REST API tests, so this PR should create such tests,
is expected that each resource will be have some basic tests.

### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -77,10 +77,9 @@ jobs:
         run: |
           docker-compose -f docker-compose.yml -f docker-compose.dev.yml -f components/serverless/docker-compose.serverless.yml up -d
           /bin/bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' ${API_ABOUT_PAGE})" != "401" ]]; do sleep 5; done'
-          cat tests/rest_api/assets/cvat_db.sql | docker exec -i cvat_db psql -q -U root -d cvat
-          cat tests/rest_api/assets/cvat_data.tar.bz2 | docker run --rm -i --volumes-from cvat ubuntu tar -xj --strip 3 -C /home/django/data
           pip3 install --user -r tests/rest_api/requirements.txt
-          pytest tests/rest_api/
+          pytest tests/rest_api/ -k 'Get'
```
Comment (nmanovic) [inline .github/workflows/main.yml:81]: Is the code necessary?

**Comment by nmanovic** [comment]
@kirill-sizov , I still can see `WIP` in the title.

**Comment by nmanovic** [comment]
@zhiltsov-max , @azhavoro , @Marishka17 , @IRDonch , 

Don't hesitate to comment. We will address them in next PRs. I have merged the PR because it is really critical.

File: tests/rest_api/conftest.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+@pytest.fixture(scope='module')
+def users_by_name(users):
+    return {user['username']: user for user in users}
+
+@pytest.fixture(scope='module')
+def find_users(test_db):
+    def find(**kwargs):
+        assert len(kwargs) > 0
+        assert any(kwargs)
```
Comment (zhiltsov-max) [inline tests/rest_api/conftest.py:80]: Probably, you need to check values instead of keys.

**Pull request merged**